.github/workflows/check.yml

```diff
@@ -243,6 +243,12 @@ jobs:
               fi
             fi
 
+            filename=$(basename $f)
```

Comment:
This check is more efficent just after file type checking like .c or .cpp.

Reply:
This behavior modifies the check results for files not present in cubrid/CMakeLists.txt, converting the check result from FAIL to PASS. If we move it as you suggested, it will create a larger condition statement structure wrapping the check section. Are you suggesting this behavior to skip the check altogether?